UI.AutomationTests/UI.AutomationTests.csproj

```diff
@@ -8,8 +8,9 @@
     </PropertyGroup>
 
     <ItemGroup>
-        <PackageReference Include="BookingsApi.Client" Version="3.0.5" />
-        <PackageReference Include="ExtentReports" Version="4.1.0" />
+        <PackageReference Include="BookingsApi.Client" Version="3.1.2-pr-0919-0015" />
```

Comment:
Need to update to latest package

Reply:
thanks, QA should hopefully wrap up soon